recipes/mlapack/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.8.1" %}
+{% set revision = "6b8193bac747fc21362253df1e7c7dd50bcb5ff1" %}
+
+package:
+  name: mlapack 
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/nakatamaho/mplapack
+  git_rev: {{ revision }}
+  md5: unused
+  patches:
+      - mlapack.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - autoconf
+    - automake
+    - gmp
+    - mpfr
+    - mpc
+    - qd
+  run:
+    - gmp
+    - mpfr
+    - mpc
+    - qd
+
+test:
+  commands:
```

Comment:
Tests are missing.

Reply:
can you please use some test from here: https://github.com/conda-forge/lapack-feedstock/blob/master/recipe/meta.yaml#L158